### ISSUE TITLE
Not working on iOS 9

### ISSUE THREAD

**Issue by jcesarmobile** [phonegap/phonegap-plugin-push#2437]
### Expected Behaviour
To work on iOS 9 or to be documented that it doesn't work on iOS 9
### Actual Behaviour
Doesn't work on iOS 9 (not even ask for push permission)
### Reproduce Scenario (including but not limited to)
Current master uses UNUserNotificationCenter, which is only available for iOS 10+, but doesn't seem to have code for older versions.
Cordova still supports iOS 9, so I expect the plugin to also support it or at least document that it doesn't.
#### Steps to Reproduce 
Use this sample app but with master version of the plugin and run on iOS 9 device
https://github.com/jcesarmobile/phonegap-plugin-push-sample
#### Platform and Version (eg. Android 5.0 or iOS 9.2.1)
iOS 9

#### Cordova CLI version and cordova platform version

    cordova --version                                   8.0.0
    cordova platform version ios                    4.5.4

#### Plugin version

    current master (appears as 2.2.3)

#### Sample Push Data Payload
not relevant
#### Sample Code that illustrates the problem
https://github.com/jcesarmobile/phonegap-plugin-push-sample

**Comment by macdonst** [comment]
@jcesarmobile Yeah, the latest commits to master will end up in a 3.0.0 version because of the iOS 10+ requirement. I'll branch off a v2.x off of the last 2.2.3 commit in case we need to do bug fixes. Supporting iOS 10, 11 and soon 12 should be good enough.

**Comment by richardshergold** [comment]
@macdonst what version of the plugin do we need to be using in order to support iOS 9?  My testers are reporting that my app fails to even install on iOS9 so I am wondering if this plugin is the issue?

Using version 2.2.3 with Cordova-IOS 4.5.5 (an Ionic app)

We used version 1 of the Plugin originally but because we migrated from GCM to FCM on the Android side we need to use version 2.  We still use APNS on the iOS side - should version 2 of the plugin support iOS9 ?

**Comment by richardshergold** [comment]
@macdonst I just found our install problem had nothing to do with the plugin as an app without the plugin would not install either.  However, I would like conformation of whether version 2.2.3 of the plugin does support iOS 9?  thanks

**Comment by richardshergold** [comment]
Thanks @jcesarmobile that's good to know.

**Comment by ppetree** [comment]
Was this ever resolved?

**Comment by jcesarmobile** [comment]
3.x is not released yet, but won’t support iOS 9.
2.x works fine, so there is nothing to resolve, other than documenting that iOS 9 is not supported before doing the release
